tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "esModuleInterop": true,
     "resolveJsonModule": true,
     "moduleResolution": "node",
-    "allowImportingTsExtensions": true
+    "allowImportingTsExtensions": false
```

Comment:
vscode me that not combining that with a bunch of other options makes it false anyway.